src/ios/CFCallNumber.m

```diff
@@ -1,34 +1,24 @@
 #import <Cordova/CDVPlugin.h>
 #import "CFCallNumber.h"
-
+ 
 @implementation CFCallNumber
-
+ 
 - (void) callNumber:(CDVInvokedUrlCommand*)command {
-
+ 
     [self.commandDelegate runInBackground:^{
-
+ 
         CDVPluginResult* pluginResult = nil;
         NSString* number = [command.arguments objectAtIndex:0];
-        number = [number stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
-
-        if( ! [number hasPrefix:@"tel:"]){
-            number =  [NSString stringWithFormat:@"tel:%@", number];
-        }
-
-        if(![[UIApplication sharedApplication] canOpenURL:[NSURL URLWithString:number]]) {
```

Comment:
Is this iOS error being looked at or should removing this error handling part become the workaround if calling from iOS is needed ?